src/components/setup_component/setup.sh

```diff
@@ -127,7 +127,15 @@ setup_imi() {
         gridDir="${Res}_${NestedRegion}"
     fi
 
+    # Clone version 14.2.1 of GCClassic
     # Define path to GEOS-Chem run directory files
+    cd "${InversionPath}"
+    rm -rf GCClassic
+    git clone https://github.com/geoschem/GCClassic.git
+    cd GCClassic
+    git checkout dac5a54 # most recent dev/14.2.1 @ 1 Sep 2023 12:44 PM (update this once 14.2.1 officially released)
+    git submodule update --init --recursive
+    cd ..
```

Comment:
Thanks Nick, I would prefer if there was a check to see if the GCClassic directory is present, rather than deleting and redownloading every time. This allows users more flexibility to modify GEOS-Chem. I think wrapping the above in `if [ ! -d "GCClassic" ]; then` would be sufficient.

Reply:
@nicholasbalasus I'm not sure a warning would be good enough here because it might get overlooked. In order to ensure the correct version of GCClassic is used, we should check for the version number in `GCClassic/CMakeLists.txt`. In that file, you should see the text `VERSION 14.2.1`. If it doesn't match the expected version then the setup script can either stop with an error (along the lines of the message you have above) or preferably print a message telling the user GCClassic is being updated to X.Y.Z and automatically update the GCClassic repo and submodules.

Also, rather than checking out a specific commit, I would recommend using the version tag (e.g. `git checkout 14.1.1`). We don't recommend pegging it to a development version in general, but if you must then you should use the branch name instead of the commit ID (e.g. `git checkout dev/14.2.1`). We can change it to use the tag once the version is officially released/tagged.